api/v1alpha1/clusterextension_types.go

```diff
@@ -110,7 +110,6 @@ const (
 	ReasonUnpackPending = "UnpackPending"
 	ReasonUnpackSuccess = "UnpackSuccess"
 	ReasonUnpackFailed  = "UnpackFailed"
```

Comment:
I think `ReasonUnpack[Pending|Failed|Success]` can go away too?

Reply:
Success and Failed are still needed, so that we set the right status when we unpack [successfully](https://github.com/operator-framework/operator-controller/blob/3fa711f9ac986a0269cf500c62b8b6a495076070/internal/controllers/clusterextension_controller.go#L259) and [error](https://github.com/operator-framework/operator-controller/blob/3fa711f9ac986a0269cf500c62b8b6a495076070/internal/controllers/clusterextension_controller.go#L239) out while unpacking.

Reply:
👍